src/containers/BasicInfoContainer.tsx

```diff
@@ -45,8 +57,10 @@ export const BasicInfoContainer: ExperimentModule<BasicInfoContainerState> = ({
       return (
         <DeviceInfoScreen
           dob={mod.dob}
+          shouldCollectDob={mod.collectDateOfBirth}
+          shouldCollectGender={mod.collectGender}
           gender={mod.gender}
-          genders={mod.genders}
+          genders={mod.genders ?? DEFAULT_GENDERS}
```

Comment:
The list of gender choices can be found here: https://github.com/flare-kcl/flare-portal/blob/68e45aa450a4a907703560755c89a8c61669b086/flare_portal/experiments/models/data.py#L146

It might be less work to just hard code them here as they're unlikely to change. They're hardcoded on the portal side too so I think this should be OK. Just leave a note that when updating the options, they should update the the one on the portal side too. I'll do the same on the portal code base.